app/src/main/java/fr/free/nrw/commons/WelcomeActivity.java

```diff
@@ -50,10 +48,17 @@ public void onCreate(Bundle savedInstanceState) {
             findViewById(R.id.finishTutorialButton).setVisibility(View.VISIBLE);
         }
 
-        ButterKnife.bind(this);
+        binding = ActivityWelcomeBinding.inflate(getLayoutInflater());
+        final View view = binding.getRoot();
+        setContentView(view);
+
+        pager = binding.welcomePager;
+        indicator = binding.welcomePagerIndicator;
 
         pager.setAdapter(adapter);
         indicator.setViewPager(pager);
+
+        pager.setOnClickListener(this::finishTutorial);
```

Comment:
The original code was setting a click listener on `R.id.finishTutorialButton` - you are binding the wrong component here.

If you use a lambda, rather than a method reference, it will let you roll back the addition of the "view" parameter, and the change in the view pager also.

Reply:
```
binding.finishTutorialButton.setOnClickListener(v -> finishTutorial());
```